<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeNavigation</class>
 <widget class="QWidget" name="NodeNavigation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>66</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonBack">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Go Back</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../libres/resources/icons.qrc">
       <normaloff>:/icons/app/arrow_left.svg</normaloff>:/icons/app/arrow_left.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>NodeNavigation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DebuggerButton" name="pushButtonDebugger">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Open Debugger</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../libres/resources/icons.qrc">
       <normaloff>:/icons/app/window/development.svg</normaloff>:/icons/app/window/development.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonMenu">
     <property name="minimumSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Main Menu</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../libres/resources/icons.qrc">
       <normaloff>:/icons/app/menu.svg</normaloff>:/icons/app/menu.svg</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DebuggerButton</class>
   <extends>QPushButton</extends>
   <header location="global">uptime/DebuggerButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../libres/resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>NodeNavigation</receiver>
   <slot>onNavigateBack()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMenu</sender>
   <signal>clicked()</signal>
   <receiver>NodeNavigation</receiver>
   <slot>onOpenMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNavigateBack()</slot>
  <slot>onOpenMenu()</slot>
 </slots>
</ui>
